packages/connect-ui/src/components/Notification.tsx

```diff
@@ -134,9 +134,9 @@ export const BackupNotification = () => (
 export const BridgeUpdateNotification = () => (
     <Notification
         variant="warning"
-        header="New bridge is available"
-        body=""
-        cta={{ desc: 'Update bridge', url: SUITE_BRIDGE_URL }}
+        header="Current bridge is outdated"
+        body="Trezor Bridge is no longer supported. Switch to Trezor Suite desktop for the best experience."
+        cta={{ desc: 'Install Trezor Suite', url: SUITE_URL }}
```

Comment:
note to me: check conditions when this is rendered.

Reply:
It seems to be checking the version from here https://connect.trezor.io/9/data/bridge/releases.json

Reply:
yep, this is there because of some ancient bridge versions, right. 

this might be, maybe, the place to start experimenting with some copy or even prompt to uninstall.